dev/src/components/__tests__/CourseCreationInfo.test.tsx

```diff
@@ -0,0 +1,20 @@
+import CourseCreationInfo from "../admin/CourseCreationInfo";
+
+import {configure, shallow} from 'enzyme'
```

Comment:
Please use consist quote type; looks like most places use `"` instead of `'`. Same applies to lines 12 and 17.